Formula/linthesia.rb

```diff
@@ -0,0 +1,19 @@
+class Linthesia < Formula
+  desc "Synthesia-like software for Linux"
+  homepage "http://linthesia.sourceforge.net/"
+  url "https://github.com/linthesia/linthesia/archive/refs/tags/0.8.0.tar.gz"
+  sha256 "0bbfeb30909fbd3a3b33bc45e8f7e75944ba6b2135a627d5729f4bda50540027"
+  license "GPL-2.0"
```

Comment:
`GPL-2.0` is deprecated. Please determine whether this is `GPL-2.0-only` or `GPL-2.0-or-later` and use the one that's appropriate.